packages/manager/apps/web/client/app/hosting/multisite/git-integration/association/git-association.service.js

```diff
@@ -34,6 +34,19 @@ export default class HostingMultisiteGitAssociationService {
     });
   }
 
+  getVcsInformations(serviceName) {
+    let websiteId = null;
+    return this.$http
+      .get(`/hosting/web/${serviceName}/website`)
+      .then(({ data: [id] }) => {
+        websiteId = id;
+        return this.$http.get(`/hosting/web/${serviceName}/website/${id}`);
+      })
+      .then(({ data: { vcsUrl, vcsBranch } }) => {
+        return { repositoryUrl: vcsUrl, branchName: vcsBranch, id: websiteId };
+      });
+  }
```

Comment:
/hosting/web/${serviceName}/website will provide some id because you didn't filter by path. And it should be an iceberg call for avoid cascade of call.

Reply:
Will replace with iceberg here then, thks!